player/public/coreplugins/org.ekstep.pdfrenderer-1.0/renderer/plugin.js

```diff
@@ -24,6 +25,7 @@ org.ekstep.contentrenderer.baseLauncher.extend({
         console.info('PDF Renderer init', manifestData)
         EkstepRendererAPI.addEventListener(this._constants.events.launchEvent, this.start, this);
         this._manifest = manifestData;
+        this._asset_arrow_path = org.ekstep.pluginframework.pluginManager.resolvePluginResource(this._manifest.id, this._manifest.ver, "renderer/assets/arrow-right.svg");
```

Comment:
Move to specific function. No need to store _asset_arrow_path  in instance level